File: RESTful Samples/Java/src/client/ReportXlsx.java
package client;

import java.io.InputStream;
import java.io.OutputStream;
import java.net.URL;

/**
 * Created by Bassem on 4/12/2015.
 */
public class ReportXlsx extends Report {
    public ReportXlsx(URL baseUrl, InputStream template, OutputStream report) {
        super(baseUrl, template, report);
    }

    public ReportXlsx(URL baseUrl, InputStream template) {
        super(baseUrl, template);
    }

    @Override
    public String outputFormat() {
        return "xlsx";
    }
}

File: RESTful Samples/RESTfulEngine_RubyClient/ReportXlsx.rb
# "THE BEER-WARE LICENSE" 
# As long as you retain this notice you can do whatever you want with this 
# stuff. If you meet an employee from Windward some day, and you think this
# stuff is worth it, you can buy them a beer in return. Windward Studios

require_relative 'report'
class ReportXlsx < Report
	def initialize uri, template, report = nil
		@OutputFormat = :xlsx
		super
	end
end